### PR TITLE
Build translator acknowledgement page

### PR DESCRIPTION
**Pull request by corwintines**
## Description
Build out the Translation acknowledgement page, and contributor listing page. 

New Pages:
[/contributing/translation-program/acknowledgements/](https://deploy-preview-4571--ethereumorg.netlify.app/en/contributing/translation-program/acknowledgements/)
[/contributing/translation-program/contributors/](https://deploy-preview-4571--ethereumorg.netlify.app/en/contributing/translation-program/contributors/)

### CODE REVIEW

File: src/components/TranslationLeaderboard.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+import React, { useState } from "react"
+import styled from "styled-components"
+
+// Components
+import Emoji from "./Emoji"
+import { ButtonSecondary } from "./SharedStyledComponents"
+import Translation from "./Translation"
+
+// Data
+import monthData from "../data/translation_reports/month_data.json"
```
Comment (samajammin) [inline src/components/TranslationLeaderboard.js:11]: Minor nitpick & I don't believe this is actually stated anywhere but we've used dashes vs. underscores for directory & file names.

File: src/content/contributing/translation-program/index.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ If you want to get involved and help us grow the global Ethereum community by tr
   <a href="https://ethereum.org/en/contributing/#poap">More on POAPs</a>
 </InfoBanner>
 
+<br/>
+
+<InfoBanner shouldCenter emoji=":tada:">
+  <a href="https://ethereum.org/en/contributing/translation-program/acknowledgements/">Translator Acknowledgements!</a>
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:28]: I'm noticing we use this incorrectly elsewhere as well but this should use the Link component so we can use relative links here vs. hardcoding the production link. i.e.

`<Link to="contributing/translation-program/acknowledgements/">Translator Acknowledgements!</Link>`

File: src/intl/en/common.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "1inch-logo": "1inch logo",
   "aave-logo": "Aave logo",
+  "acknowledgements": "Acknowledgements",
```
Comment (samajammin) [inline src/intl/en/common.json:4]: Doesn't seem like we use this anywhere?

**Comment by samajammin** [comment]
This looks a bit awkward. Let's chat through alternatives here:
Image 2021-11-29 at 5 38 05 PM

File: src/intl/en/page-contributing-translation-program-acknowledgements.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  "page-contributing-translation-program-acknowledgements-acknowledgement-page-title": "Acknowledgement Page",
```
Comment (samajammin) [inline src/intl/en/page-contributing-translation-program-acknowledgements.json:2]: "Acknowledgement Page" feels like a weird page title to me. Maybe something more "human", like "Acknowledging our contributors"?

File: src/components/TranslationLeaderboard.js [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+                  </div>
+                </TextContainer>
+                <LangaugeContainer>
+                  {languages[0]?.name || ""}
+                </LangaugeContainer>
+                <WordsContainer>
+                  <StyledEmoji
+                    ml={"1rem"}
+                    mr={"1rem"}
+                    size={1.5}
```
Comment (samajammin) [inline src/components/TranslationLeaderboard.js:221]: The sizing here doesn't appear to be enforced - I'm seeing a range of sizes that looks to vary depending on the size of the `translated` number:
Image 2021-11-29 at 7 47 00 PM

File: src/components/TranslationLeaderboard.js [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+              emoji = ":trophy:"
+            } else if (idx === 1) {
+              emoji = ":2nd_place_medal:"
+            } else if (idx === 2) {
+              emoji = ":3rd_place_medal:"
+            }
+            return (
+              <Item key={idx}>
+                {emoji ? (
+                  <Emoji mr={"1rem"} text={emoji} />
```
Comment (samajammin) [inline src/components/TranslationLeaderboard.js:203]: These trophy emojis feel way too small - can we make them look similar to the bug bounty leaderboard?
Image 2021-11-29 at 7 52 11 PM

Comment (samajammin) [reply src/components/TranslationLeaderboard.js:203]: I know we're listing more results but in general I feel like this table could use a bit more breathing room, similar to size & spacing of the bug bounty leaderboard rows.

File: src/components/TranslationLeaderboard.js [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+
+const ButtonRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1rem;
+  flex-wrap: wrap;
+`
+
+const ButtonSecondaryStyled = styled(ButtonSecondary)`
```
Comment (samajammin) [inline src/components/TranslationLeaderboard.js:96]: I like the look of these buttons but they seem too close to the table & overall the spacing feels tight:
Image 2021-11-29 at 7 56 09 PM

I feel like we could work off existing elements, e.g. the pills of the dapps page:
Image 2021-11-29 at 7 55 54 PM

Comment (samajammin) [reply src/components/TranslationLeaderboard.js:96]: Really whenever possible I'd encourage us to re-use similar elements for consistency (& ideally sharing the styles in our code).

File: src/intl/en/common.json (hunk 3 above)
Comment (corwintines) [reply src/intl/en/common.json:4]: The breadcrumbs use this

**Review by wackerow — COMMENTED** [review]
This is looking awesome! Nice job @corwintines 🤩 Left a couple code suggestions... one is just fixing the alignment of text, the other is up to the collective, if we're down for larger avatars with the current layout.

Looking forward to getting this up!

**Comment by minimalsm** [comment]
I like the increased size of the medal/trophy emojis but we should fix the alignment between emojis and regular numbers.

Current:
Screenshot 2021-12-01 at 13 11 46

Suggested:
Screenshot 2021-12-01 at 13 15 56

**Comment by minimalsm** [comment]
Given that the translation leaderboard is the main feature of this page I'm wondering if giving it more space on the page might make more sense. We could easily add the accompanying block of text below it.

Screenshot 2021-12-01 at 13 21 46

**Comment by samajammin** [comment]
> Given that the translation leaderboard is the main feature of this page I'm wondering if giving it more space on the page might make more sense. We could easily add the accompanying block of text below it.

I agree with @minimalsm - we could easily consolidate that text and/or move it below.

**Comment by samajammin** [comment]
Noticed some copy changes weren't displaying in the preview - just tried clearing the Netlify cache & redeploying.

**Comment by samajammin** [comment]
I still think we should merge these callouts - let's quickly chat through copy tomorrow:
Image 2021-12-01 at 9 42 58 PM

**Review by samajammin — CHANGES_REQUESTED** [review]
Looking great! I think it's just about there. Commented with couple change suggestions we can chat through tomorrow.

**Comment by minimalsm** [comment]
> I like the increased size of the medal/trophy emojis but we should fix the alignment between emojis and regular numbers.
> 

Sorry, I don't think this was clear. I've tried to illustrate the problem better with the image below.

Screenshot 2021-12-02 at 13 45 42

File: src/pages/contributing/translation-program/contributors.js [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    grid-template-columns: 1fr 1fr;
+  }
+`
+
+const Contributors = ({ location }) => {
+  return (
+    <Page>
```
Comment (pettinarip) [inline src/pages/contributing/translation-program/contributors.js:38]: Shouldn't we have a `PageMetadata` here as well? I've loaded the page and I don't see a title in the browser tab.

File: src/pages/contributing/translation-program/contributors.js [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+                item.user.username !== "ethdotorg" &&
+                !item.user.username.includes("LQS") &&
+                !item.user.username.includes("REMOVED_USER")
+            )
+            .map(({ user }) => user.username)
+            .sort((user1, user2) =>
+              user1.toLowerCase().localeCompare(user2.toLowerCase())
+            )
+            .map((user) => {
+              return <ListItem>{user}</ListItem>
```
Comment (pettinarip) [inline src/pages/contributing/translation-program/contributors.js:76]: Lets add a `key` here `<ListItem key={user}>{user}</ListItem>`

**Review by pettinarip — COMMENTED** [review]
GJ! looking great @corwintines. I've added a couple of minor comments.

**Review by pettinarip — APPROVED** [review]

**Review by Aivolt1 — APPROVED** [review]

**Review by samajammin — CHANGES_REQUESTED** [review]
This looks terrific @corwintines!

Only a couple small mobile issues I'd like us to clean up before merging:

Not a great word wrap in the title on mobile. Perhaps we reduce the header size on mobile?
95C61D6D-B20C-4F51-A839-1527F990540D_1_105_c

Image is cropped on mobile:
B8A406EB-070C-40E6-A265-DCF348D5316F_1_105_c

**Review by samajammin — APPROVED** [review]
🎉

**Pull request merged**